Improve Helm experience

WyriHaximus opened the pull request:
Since the deprecation of the `stable` Helm chart repository by Helm, chaoskube's chart is one of the charts I've been waiting for to be ported.

Initially, when that happened I didn't get it to work and didn't look too deep into it. Now that I'm building a home cluster it came up again,
and after looking into it properly there are a few things that came up:

- [X] The project's readme still lists the old `stable` Helm chart repository.
- [X] The chart's readme lists a non-existent (?) Helm chart repository
- [X] Has default values for arguments causing issues for some of those when empty
- [X] Missing `v` for the default tag
- [X] Using a non-amd64 OCI image on a raspberry pi was a bit of a search to find out and get working

P.S. This PR only aims to make it work out of the box again and contains some breaking changes causing the major version bump.

colinbut commented on README.md:
````diff
@@ -45,7 +45,8 @@ INFO[4804] terminating pod          name=nginx-701339712-51nt8 namespace=chaosku
 ```console
-$ helm install stable/chaoskube
+$ helm repo add chaoskube https://linki.github.io/chaoskube/
+$ helm install chaoskube chaoskube/chaoskube --atomic --namespace=chaoskube --create-namespace
````
The old instructions got me initially too. New instructions will be helpful for new users 👍

zwhite commented:
I was just tracking down why it is I can't get chaoskube to start in Dry Run mode, and this PR would fix it for me. Would help me a lot to get this in.

linki commented:
@WyriHaximus Thanks a lot for this PR.

Meanwhile I migrated the docker images from `quay.io/linki/chaoskube` to `ghcr.io/linki/chaoskube`. The new image on ghcr.io is a multi-arch image so we could remove the need for handling the architecture.

@WyriHaximus Would you mind updating the image in this PR and removing the architecture related part?

WyriHaximus commented:
> @WyriHaximus Thanks a lot for this PR.

@linki :+1: , also rebased this PR on the latest master

> Meanwhile I migrated the docker images from `quay.io/linki/chaoskube` to `ghcr.io/linki/chaoskube`. The new image on ghcr.io is a multi-arch image so we could remove the need for handling the architecture.

Awesome, updated the chart with the new repository and confirmed it runs on `arm64` :tada: .
 
> @WyriHaximus Would you mind updating the image in this PR and removing the architecture related part?

:heavy_check_mark:

linki commented:
@WyriHaximus Thanks a lot! Sorry it took so long.

WyriHaximus commented:
@linki Cheers, and no worries. Life is always more important to work :).
